src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/regler/vilkår/UtgifterOvernattingRegel.kt

```diff
@@ -25,5 +45,15 @@ class UtgifterOvernattingRegel :
                         SvarId.NEI to IKKE_OPPFYLT_MED_PÅKREVD_BEGRUNNELSE,
                     ),
             )
+        private val HØYERE_UTGIFTER_HELSEMESSIG_ÅRSAKER =
+            RegelSteg(
+                regelId = RegelId.HØYERE_UTGIFTER_HELSEMESSIG_ÅRSAKER,
+                erHovedregel = true,
+                svarMapping =
+                    mapOf(
+                        SvarId.JA to OPPFYLT_MED_VALGFRI_BEGRUNNELSE,
+                        SvarId.NEI to IKKE_OPPFYLT_MED_PÅKREVD_BEGRUNNELSE,
+                    ),
```

Comment:
```suggestion
                     mapOf(
                        SvarId.JA to OPPFYLT_MED_PÅKREVD_BEGRUNNELSE,
                        SvarId.NEI to OPPFYLT_MED_VALGFRI_BEGRUNNELSE,
                    ),
```

Denne skal uansett være oppfylt. Hvis ja må saksbehandler legge til begrunnelse. Hvis nei er begrunnelse valgfri. 

Senere skal vi bruke dette vilkåret i beregningen for å finne ut hvor mye penger søker har rett på.